FBC/FunctionalEntities/NorthAmericanEntities/USRegulatoryAgencies.rdf

```diff
@@ -694,27 +706,27 @@
 				<owl:hasValue rdf:resource="&fibo-fbc-fct-usjrga;FederalReserveRegulatoryAgencyAndCentralBank"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label>Federal Reserve System (FRS) member bank</rdfs:label>
+		<rdfs:label>FRS member bank</rdfs:label>
```

Comment:
If it's a bank why does it not subclass fibo-fbc-fct-fse;Bank?

Reply:
The term 'bank' as used by the Fed, includes some other institutions like investment banks, which are not the same as FDIC insured banks, so that's why it is a subclass of financial institution rather than bank.

Reply:
I think that would be useful to add as an explanatory note, together with something about any constraints the Fed has on what distributions can be classed as Banks. Maybe we should have a separate class to represent that, since it seems important. Or maybe Government Insured Bank should be the separate class (not using "FDIC" which is  jurisdiction-specific).

Reply:
Created #890

Reply:
Raised #890